<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Color_recovery</class>
 <widget class="QDialog" name="Color_recovery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color_recovery</string>
  </property>
  <widget class="QPushButton" name="file_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>사진 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>창 닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="eccv16_image">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="siggraph17_image">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="eccv16_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>저장하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="siggraph17_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>저장하기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
